src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -3723,8 +3723,12 @@ namespace InternalHeatGains {
                       Format_723,
                       "People",
                       "Number of People {},People/Floor Area {person/m2},Floor Area per person {m2/person},Fraction Radiant,Fraction "
-                      "Convected,Sensible Fraction Calculation,Activity level,ASHRAE 55 Warnings,Carbon Dioxide Generation Rate,Nominal Minimum "
-                      "Number of People,Nominal Maximum Number of People");
+                      "Convected,Sensible Fraction Calculation,Activity level,ASHRAE 55 Warnings,Carbon Dioxide Generation Rate,"
+                      "Minimum Number of People for All Day Types,Maximum Number of People for All Day Types,"
+                      "Minimum Number of People for Weekdays, Maximum Number of People for Weekdays, "
+                      "Minimum Number of People for Weekends/Holidays, Maximum Number of People for Weekends /Holidays,"
```

Comment:
The extra space in the maximum column (Weekends /Holidays) actually has a good result on the column heading.

![image](https://github.com/NREL/EnergyPlus/assets/5778156/71c29688-7a4a-477e-904f-b48e883111db)